runeq/resources/stream_metadata.py

```diff
@@ -817,6 +791,16 @@ def get_stream_metadata(
         )
         stream_set.add(stream)
 
+        try:
+            seen_stream_ids.remove(stream.id)
+        except KeyError:
+            pass
+
+    if len(seen_stream_ids) > 0:
+        raise RuneError(
+            f"1+ stream ID(s) not found: {','.join(seen_stream_ids)}"
+        )
```

Comment:
Nice addition!

Reply:
haha it was to match the docstring you wrote ;-) it was a good idea!!